.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.Authenticator;
+import java.net.PasswordAuthentication;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.util.Properties;
+
+public final class MavenWrapperDownloader
```

Comment:
So we do want the downloader and not the jar in git? I asking because of the following discussion: https://github.com/quarkusio/quarkus-quickstarts/pull/1138

Reply:
No problem, let me change it to jar.

Reply:
Done https://github.com/quarkiverse/quarkus-cxf/pull/487/commits/83d1f7c36bdfca6f417f30ee42355e9356edc2e9#diff-16ac94b87c2656c6987baba47a65cad02d58a1b27f31f1bc9133746ddc9780c4